docs/DeveloperGuide.md

```diff
@@ -288,30 +288,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `BizBook` and the **Actor** is the `user`, unless specified otherwise)
```

Comment:
Since `System` is an alias of `BizBook` and `Actor` is an alias of `user`, then the following text below should use that convention.

Reply:
Sorry, but I'm not sure what you mean by this. Are you saying i should replace user with `user`?

Reply:
Oh I'm saying that you should replace
- `user` with Actor
- `BizBook` with System